packages/vscode-extension/src/webview/views/PreviewView.tsx

```diff
@@ -84,7 +85,11 @@ function PreviewView() {
     useProject();
   const { showDismissableError } = useUtils();
 
-  const [isInspecting, setIsInspecting] = useState(false);
+  // The inspector can be in one of the following states:
+  // "disabled" - inspector is not active, device is interactable
+  // "inspecting" - inspector is active and inspecting elements on hover
+  // "selected" - no more hover, but highlight is active on an element
```

Comment:
If we change `InspectorState` to an enum, we can have this as a doc comment on the enum variants.